File: App/app/src/main/java/com/alexz/messenger/app/ui/common/contentgridlayout/ContentClickListener.java
package com.alexz.messenger.app.ui.common.contentgridlayout;

import android.widget.ImageView;

import com.alexz.messenger.app.data.model.interfaces.IMediaContent;
import com.google.android.exoplayer2.ui.PlayerView;

public interface ContentClickListener {

    void onImageClick(ImageView view, IMediaContent content);

    void onVideoClick(PlayerView playerView, IMediaContent content);
}

File: App/app/src/main/java/com/alexz/messenger/app/ui/common/contentgridlayout/ContentClickListener.kt
package com.alexz.messenger.app.ui.common.contentgridlayout

import android.widget.ImageView
import com.alexz.messenger.app.data.entities.interfaces.IMediaContent
import com.google.android.exoplayer2.ui.PlayerView

interface ContentClickListener {
    fun onImageClick(view: ImageView, content: IMediaContent)
    fun onVideoClick(playerView: PlayerView, content: IMediaContent)
}